### DISCUSSION TITLE
Reload Aborted/Failed to InfluxDB

### DISCUSSION THREAD

**Discussion by Chrisdup8710** [ptarmiganlabs/butler#931] [General]
Compliments of the season!

I have been running Butler for a while on our environment, and successfully capturing Windows Service data to influx, though I want to know if it is possible to store data of Aborted/Failed reloads to Influx?

I went through the config file and found there is no pertinent settings for storing failed reloads to Influx, though there are settings for sending mail alerts.

Reason for asking is we want to use grafana to generate the alerts to eventually send on to PagerDuty once implemented.

Thank in advance and good luck for the coming year!

PS. I have attached the yaml config file for your perusal.

[NewButler.txt](https://github.com/ptarmiganlabs/butler/files/13858918/NewButler.txt)

**Comment by mountaindude** [comment]
@Chrisdup8710 

Indeed, it is possible to store reload failed (and successful reloads too) data in InfluxDB.

This was added fairly recently (v9.3.0), so make sure you get the latest version of Butler (right now this is v9.4.0) for this to work.
It's documented at the [Butler documentation site](https://butler.ptarmiganlabs.com/docs/getting-started/setup/reload-alerts/alert-influxdb/).

A summary of this feature (incl sample Grafana visualisations) is available in a blog post over at [ptarmiganlabs.com](https://ptarmiganlabs.com/senseops-2023-q4-wrap-up/).

Feel free to [sign up to the newsletter](https://ptarmiganlabs.com/#/portal/signup) there to get latest blog posts to your inbox.

Teaser Grafana chart showing failed reload data:
![bnutler-grafana-failed-reload-1](https://github.com/ptarmiganlabs/butler/assets/1029262/c8fea3fb-854b-4fc9-8758-3002b9bfbddb)

Grafana chart showing number of reloads per 15-min time windows:
![image](https://github.com/ptarmiganlabs/butler/assets/1029262/5d5ed06b-e64d-41b2-93d3-04938b2aa193)
